### DISCUSSION TITLE
We built web UI for Sherlock

### DISCUSSION THREAD

**Discussion by sagarspatil** [sherlock-project/sherlock#1064] [Q&A]
My friend and I built an UI for Sherlock so you don't have to install anything and you can start searching directly on the website - https://userhunt.co/
We have included around ~150 sites. If there's more demand, we can keep adding new sites.

**Comment by hoadlck** [comment]
Nice site design. :-)

**Reply by sagarspatil** [reply]
Thank you kind sir.

**Reply by 007unicorn** [reply]
The UI isn't working for me?

**Reply by jpmortaza** [reply]
> A interface do usuário não está funcionando para mim?

It's working.

**Comment by akaye47** [comment]
I’ve just been locked out of this email bc Christine again

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Cfomodz** [comment]
Dead, can be closed?
